src/init/starship.zsh

```diff
@@ -16,6 +16,20 @@ starship_render() {
     PROMPT="$(::STARSHIP:: prompt --keymap="${KEYMAP-}" --status=$STATUS --cmd-duration=${STARSHIP_DURATION-} --jobs="$NUM_JOBS")"
 }
 
+# Defines a function `__starship_get_time` that sets the time since epoch in millis in STARSHIP_CAPTURED_TIME.
+if [[ $ZSH_VERSION == ([1-4]*) ]]; then
+    # ZSH <= 5; Does not have a built-in variable so we will rely on Starship's inbuilt time function.
+    __starship_get_time() {
+        STARSHIP_CAPTURED_TIME=$(::STARSHIP:: time)
+    }
+else
+    zmodload zsh/datetime
+    zmodload zsh/mathfunc
+    __starship_get_time() {
+        ((STARSHIP_CAPTURED_TIME = int(rint($EPOCHREALTIME * 1000))))
```

Comment:
I think this will break [`show_milliseconds`](https://starship.rs/config/#command-duration) in the  command duration module.

Reply:
The output is time since epoch in millis as an integer, the same as `starship time`. This should be a drop-in replacement. Am I missing something? I used this approach because `show_milliseconds` is an option we support for `cmd_duration`. If not, there are simpler and more-widely supported options like `$SECONDS`.

Sample Output:
```
~ ❯ starship time
1602149653499
~ ❯ ((currentTimeInEpoch = int(rint($EPOCHREALTIME * 1000)))) && echo $currentTimeInEpoch
1602149677644
```

Reply:
Sorry the `* 1000` part mislead me.